### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by jitendra-webkul**
**BUGS:**

Updated changelog

### CODE REVIEW

File: CHANGELOG for v1.x.x.md [hunk 1]
```diff
@@ -349,8 +263,6 @@
 
 * #3222 [fixed] - UI issue in event ticket booking special price date field
 
-* #3227 [fixed] - Guest coupon code cannot be used for general users
-
 * #3228 [fixed] - product videos in e-commerce
```
Comment (ghermans) [inline CHANGELOG for v1.x.x.md:266]: This is also a general question / request

File: CHANGELOG for v1.x.x.md [hunk 2]
```diff
@@ -349,8 +263,6 @@
 
 * #3222 [fixed] - UI issue in event ticket booking special price date field
 
-* #3227 [fixed] - Guest coupon code cannot be used for general users
-
 * #3228 [fixed] - product videos in e-commerce 
 
 * #3230 [fixed] - Redirect to my profile from checkout page after login
```
Comment (ghermans) [inline CHANGELOG for v1.x.x.md:268]: This was a invalid bug report.

Comment (jitendra-webkul) [reply CHANGELOG for v1.x.x.md:268]: done

File: CHANGELOG for v1.x.x.md (hunk 1 above)
Comment (jitendra-webkul) [reply CHANGELOG for v1.x.x.md:266]: Done

**Pull request merged**